extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -20,7 +20,12 @@ import {dev,user} from '../../../src/log';
 import {removeElement} from '../../../src/dom';
 import {toggle} from '../../../src/style';
 import {listenOnce} from '../../../src/event-helper';
+import {startsWith} from '../../../src/string';
+import {setStyle} from '../../../src/style';
+import {isExperimentOn} from '../../../src/experiments';
 
+/** @private @const {string} */
+const TAG = 'amp-sticky-ad-v1';
```

Comment:
name this `UX_EXPERIMENT`, unlike other cases we are not sharing tag name and experiment name here.

Also maybe a more descriptive experiment name? `amp-sticky-ad-better-ux` ?

Also cleanup issue for the experiment with a comment pointing to issue #